app/services/push_notifications/broadcast.rb

```diff
@@ -0,0 +1,28 @@
+module PushNotifications
+  class Broadcast
+    def initialize(push_notifications:)
+      @push_notifications = push_notifications
+    end
+
+    def send
+      return unless push_notifications.any?
+
+      client = Exponent::Push::Client.new
+
+      client.publish(notifications)
+    end
+
+    private
+
+    attr_reader :push_notifications
+
+    def notifications
+      push_notifications.map do |push_notification|
+        {
+          'to' => push_notification.to,
+          'title' => push_notification.title
```

Comment:
does `Exponent::Push::Client` requires the keys to be strings? It'd be great if we could use symbols.

Reply:
It should be bossible :+1: